ui/packages/catalog-ui-search/package.json

```diff
@@ -26,6 +26,7 @@
     "@types/backbone.marionette": "3.3.2",
     "@types/openlayers": "4.3.11",
     "ace": "*",
+    "codice-icons": "*",
```

Comment:
❓ Does this need to be a regular non-dev dependency?

Reply:
Yeah that would make more sense - updated 👍